stl/inc/chrono

```diff
@@ -6040,58 +6052,49 @@ struct formatter<_CHRONO utc_time<_Duration>, _CharT> {
     }

```

Comment:
Note that it doesn't accurately format time points during leap second insertion.
No changes requested for the scope of this PR.

Reply:
Card added to the Chrono Project for these formatters, thanks.